### PR TITLE
Fix a bug on messages generated into the default package.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

**Comment by JakeWharton** [comment]
lgtm

File: wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java [hunk 1]
```diff
@@ -1023,7 +1023,9 @@ private String adapterString(ProtoType type) {
   }
 
   private String reflectionName(ClassName className) {
```
Comment (JakeWharton) [inline wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java]: Seems like this could be upstreamed into a member of `ClassName`.

Comment (JakeWharton) [inline wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java]: `toString()` + `toCanonicalString()` or whatever

Comment (swankjesse) [inline wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java]: Yeah, we want that and something for members.

**Pull request merged**